libc/src/__support/CPP/type_traits/is_trivially_destructible.h

```diff
@@ -16,7 +16,7 @@
 namespace LIBC_NAMESPACE::cpp {
 
 // is_trivially_destructible
-#if LIBC_HAS_BUILTIN(__is_trivially_destructible)
+#if __has_builtin(__is_trivially_destructible)
```

Comment:
We no longer need to include src/__support/macros/config.h here thanks to this change.

Reply:
Done.